controller/DeliveryRunner.php

```diff
@@ -24,6 +24,8 @@
 use oat\tao\model\theme\ThemeService;
 use oat\taoDelivery\controller\DeliveryServer;
 use oat\taoLti\models\classes\theme\LtiHeadless;
+use oat\taoProctoring\model\execution\DeliveryExecutionManagerService;
```

Comment:
There is should not be dependencies between `taoLtiDeliveryProvider` and `taoProctoring`. Looks like logic should be moved